.gitignore

```diff
@@ -60,3 +60,6 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+#Trash files in Mac
+.DS_Store
```

Comment:
Normally, that would go in a global .gitignore file. Let's keep only project-specific ignores here.